src/videojs-http-streaming.js

```diff
@@ -95,6 +95,17 @@ const simpleTypeFromSourceType = (type) => {
     return 'dash';
   }
 
+  // Denotes the special case of a pre-parsed manifest object passed in instead of the
+  // traditional source URL.
+  //
+  // See https://en.wikipedia.org/wiki/Media_type for details on specifying media types.
+  //
+  // In this case, vnd is for vendor, VHS is for this project, and the +json suffix
+  // identifies the structure of the media type.
+  if (type === 'application/vnd.vhs+json') {
+    return 'vhs-json';
```

Comment:
is it possible to know whether it is HLS or DASH so that later if you do `player.currentType()` you'll get back an HLS or DASH type?

Reply:
I guess it doesn't have to be here specifically.

Reply:
Technically the VHS JSON object is streaming format independent. For instance, we can concatenate DASH and HLS together into one VHS JSON object.